src/exportImage.py

```diff
@@ -167,3 +192,22 @@ def runPreperation(self, taskId, filename):
         self.__helper.save_update(
             taskId, Status.IN_PROGRESS.value, filename, 0, None, None, attempts+1)
         return True
+
+    def get_zoom_resolution(self, zoom_level):
+        if f'{zoom_level}' in self.__zoom_to_resolution:
+            resolution = self.__zoom_to_resolution[f'{zoom_level}']
+            return resolution
+        else:
+            raise Exception(f'No such zoom level. got: {zoom_level}')
+
+    def is_valid_zoom_for_area(self, resolution, bbox, zoom_level):
+        top_right_lat = bbox[3]
+        top_right_lon = bbox[2]
+        bottom_left_lat = bbox[1]
+        bottom_left_lon = bbox[0]
+
+        # Check if bbox width and height are at least at the resolution of a pixle at the wanted zoom level
+        return (top_right_lon - bottom_left_lon) > float(resolution) and (top_right_lat - bottom_left_lat) > float(resolution)
+
+    def is_valid_overview_factor(self, dataset, ov_factor):
+        return dataset.RasterXSize / ov_factor >= 8 and dataset.RasterYSize / ov_factor >= 8
```

Comment:
why to use get_zoom_resolution if resolution data already exists in the image data